.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
-python: 3.6
+python:
+- 3.6
+- 3.7
```

Comment:
It looks like you may need to copy a few lines from NumPy's `.travis.yml` to check Python 3.7 working:
https://github.com/numpy/numpy/blob/4f1541e1cb68beb3049a21cbdec6e3d30c2afbbb/.travis.yml#L39-L41